templates/model.txt

```diff
@@ -27,6 +33,7 @@ module.exports = mongoose.model('<%= table %>', schema, '<%= table %>');
 <%if (!hasIdColumn && !hasPrimaryKeys) {%>
   <%= modelName %>.removeAttribute('id');
 <% } %>
+  // This section contains the relationships for the `<%= modelName %>` model. See: https://docs.forestadmin.com/documentation/v/v4/reference-guide/relationships#adding-relationships.
```

Comment:
```suggestion
  // This section contains the relationships for this model. See: https://docs.forestadmin.com/documentation/v/v4/reference-guide/relationships#adding-relationships.
```

Reply:
As we said, keep it simple ;)

Reply:
This section contains the relationships OF this model. Learn more here: https://docs.forestadmin.com/documentation/v/v4/reference-guide/relationships#adding-relationships.